components/TwitterShare.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div class="TwitterShare">
+    <v-btn rounded color="#00acee" class="ShareButton" @click="twitter">
```

Comment:
```suggestion
    <v-btn
      rounded
      color="#1DA1F2"
      class="ShareButton"
      href="https://twitter.com/intent/tweet?text=Civictech%20Challenge%20Cup%20U-22&url=https://ccc2020.code4japan.org&hashtags=CCCu22"
      target="noopener noreferrer"
    >
```

一応Twitterの色は `#1DA1F2` ということになってる（[この資料の p.8 参照](https://about.twitter.com/content/dam/about-twitter/company/brand-resources/en_us/Twitter_Brand_Guidelines_V2_0.pdf)）